kafka/source/pkg/reconciler/kafkasource.go

```diff
@@ -75,7 +76,8 @@ type Reconciler struct {
 	receiveAdapterImage string
 	eventTypeLister     eventinglisters.EventTypeLister
 	kafkaLister         listers.KafkaSourceLister
-	sinkReconciler      *duck.SinkReconciler
+	sinkResolver     	*resolver.URIResolver
+	sinkReconciler   	*duck.SinkReconciler
```

Comment:
I don't think this should be there anymore? sinkResolver should be the only thing you need?